components/admin_console/admin_definition.jsx

```diff
@@ -6040,63 +6040,6 @@ const AdminDefinition = {
                         isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.EXPERIMENTAL.FEATURES)),
                         isHidden: it.configIsFalse('FeatureFlags', 'CollapsedThreads'),
                     },
-                    {
```

Comment:
Shouldn't we also remove `ServiceSettings.EnableLegacySidebar` from admin_definition? (Line 6005)